src/goDeclaration.ts

```diff
@@ -91,11 +93,17 @@ function definitionLocation_godef(input: GoDefinitionInput, token: vscode.Cancel
 	if (token) {
 		token.onCancellationRequested(() => killProcess(p));
 	}
-	const cwd = getWorkspaceFolderPath(input.document.uri);
+
+	let cwd: string;
+	if (input.isMod) {
+		cwd = input.modPath;
+	} else {
+		cwd = getWorkspaceFolderPath(input.document.uri);
+	}
 
 	return new Promise<GoDefinitionInformation>((resolve, reject) => {
 		// Spawn `godef` process
-		p = cp.execFile(godefPath, ['-t', '-i', '-f', input.document.fileName, '-o', offset.toString()], { env, cwd }, (err, stdout, stderr) => {
+		p = cp.execFile(godefPath, ['-t', '-f', input.document.fileName, '-o', offset.toString()], { env, cwd }, (err, stdout, stderr) => {
```

Comment:
What's the reason for removing the `-i` flag?
We had that in place to support the Go to definition feature on unsaved changes in the file